README.md

```diff
@@ -102,12 +102,6 @@ To create the test database, run:
 createdb cfdm_unit_test
 ```
 
-Create the required roles in the test database by running:
-
-```
-psql <<db-connection-parameters>> -f data/create_roles.sql
-```
-
```

Comment:
Love that we can continue to simplify things! :-)

Reply:
Do we need to add documentation for the new `$SQLA_SAMPLE_DB_CONN` environment variable, though, so folks are aware of it and how to use it?

Reply:
We also need to update the documentation on how to load the sample data.  Speaking of which...  if you're starting from scratch, it's a matter of creating the database, then running migrations, then loading the data, correct?  Could we setup an `invoke` task or a `manage.py` management command that will do all of this in one shot for us?

Reply:
The issue with creating a database in one command is that you now have to specify 2 DBs - one to log into, and the other that needs to be created. Or, we have to assume that the user has a default DB that is implicit and which we derive by removing the explicit DB parameter in the SQLA connection...

Reply:
Ah, I would leave it then as just a command for everything after the database (e.g., `cfdm_test`) has been created.